frontend/src/constants/FixedQuestions.ts

```diff
@@ -0,0 +1,108 @@
+import { FormQuestion } from "../types/CampsTypes";
+
+export const fixedCamperInfoQuestions: Array<FormQuestion> = [
+  {
+    type: "Text",
+    question: "Camper First Name",
+    required: true,
+    category: "PersonalInfo",
+    id: "NONE",
```

Comment:
Missing context, is this just "NONE" as a placeholder value? Do we ever use this?

Reply:
they are placeholder values and they will never be sent to the backend as it's part of view only questions. I didn't make this `CreateFormQuestion` type because sometime we need to combine the fixed questions with the template questions (in camp creation flow) and it's easier if the `QuestionCard` component (which works with both dynamic and these fixed questions) just takes 1 prop of a consistent type (FormQuestion)